moveit_commander/src/moveit_commander/move_group.py

```diff
@@ -423,20 +423,20 @@ def clear_path_constraints(self):
         self._g.clear_path_constraints()
 
     def get_trajectory_constraints(self):
-        """ Get the actual trajectory constraints in form of a moveit_msgs.msgs.Constraints """
-        c = Constraints()
+        """ Get the actual trajectory constraints in form of a moveit_msgs.msgs.TrajectoryConstraints """
+        c = TrajectoryConstraints()
         c_str = self._g.get_trajectory_constraints()
         conversions.msg_from_string(c, c_str)
         return c
 
     def set_trajectory_constraints(self, value):
-        """ Specify the trajectory constraints to be used """
+        """ Specify the trajectory constraints to be used (database not implemented)"""
```

Comment:
```suggestion
        """ Specify the trajectory constraints to be used (setting from database is not implemented yet)"""
```
I'm not sure the docstring is the best place to keep track of this, but this should be clearer at least